### PR TITLE
Don't use set-env in the CIs

### PR DESCRIPTION
**Pull request by tczekajlo**
Resolves https://github.com/RasaHQ/rasa-x/issues/3668

### CODE REVIEW

**Comment by tczekajlo** [comment]
/modeltest

```yml
include:
 - dataset: ["Carbon Bot"]
   config: ["Sparse + DIET(bow) + ResponseSelector(bow)"]

```

File: .github/workflows/continous-integration.yml [hunk 1]
```diff
@@ -332,9 +332,9 @@ jobs:
         shell: bash
         run: |
           if [ -z "${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN }}" ]; then
-            echo ::set-env name=MERGE_TOKEN::${{ secrets.GITHUB_TOKEN }}
+            echo MERGE_TOKEN=${{ secrets.GITHUB_TOKEN }} >> $GITHUB_ENV
```
Comment (ricwo) [inline .github/workflows/continous-integration.yml:335]:
```suggestion
            echo "MERGE_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
```

File: .github/workflows/continous-integration.yml [hunk 2]
```diff
@@ -332,9 +332,9 @@ jobs:
...
           else
-            echo ::set-env name=MERGE_TOKEN::${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN }}
+            echo MERGE_TOKEN=${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN }} >> $GITHUB_ENV
```
Comment (ricwo) [inline .github/workflows/continous-integration.yml:337]:
```suggestion
            echo "MERGE_TOKEN=${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN }}" >> $GITHUB_ENV
```

**Review by ricwo — APPROVED** [review]
💯

**Pull request merged**